restconnector/RestZeqTranslator.cpp

```diff
@@ -4,35 +4,59 @@
 
 #include <restconnector/RestZeqTranslator.h>
 #include <zeq/hbp/vocabulary.h>
+#include <zeq/hbp/camera_generated.h>
+#include <zeq/hbp/request_generated.h>
 
 #include <boost/algorithm/string.hpp>
+#include <lunchbox/log.h>
+
+#include <iostream>
 
 namespace restconnector
 {
 
-RestZeqTranslator::RestZeqTranslator(){}
+RestZeqTranslator::RestZeqTranslator()
+{
+}
 
-RestZeqTranslator::~RestZeqTranslator(){}
+RestZeqTranslator::~RestZeqTranslator()
+{
+}
 
-::zeq::Event RestZeqTranslator::translate()
+::zeq::Event RestZeqTranslator::translate( const std::string& body )
 {
     //Here we should use the rest2Zeq mapper that need to be created in zeq::vocabulary
-    lunchbox::uint128_t test;
-    ::zeq::Event event = zeq::hbp::serializeRequest( test );
-    return event;
+    lunchbox::uint128_t eventType;
+    std::string zeqBody;
+
+    if( command_ == "setCamera" )
+    {
+        // This must be in the vocabulary
+        // The vacabulary is responsible for returning the
+        // zeq event
+        eventType = ::zeq::hbp::EVENT_CAMERA;
+        zeqBody = body;
+    }
+    else if( command_ == "request" )
+    {
+        //  This must be in the vocabulary
+        eventType = ::zeq::hbp::EVENT_REQUEST;
+        zeqBody = "{\"eventLow\": 0, \"eventHigh\": 0}";
```

Comment:
should be fixed/documented in zeq. Either document that empty body is invalid or handle empty w/o crash